configuration/s01_pipeline_config.json

```
@@ -1,5 +1,5 @@
 {
-  "PipelineName": "IMAQAL_COVID19",
+  "PipelineName": "imaqal_covid19_all_seasons",
```

```suggestion
  "PipelineName": "IMAQAL_COVID19_all_seasons",
```
for consistency of formatting with Coda and the traffic dashboards

---

updated thanks.